assets/pages/digital-subscription-checkout/digitalSubscriptionCheckout.scss

```diff
@@ -34,10 +34,7 @@
 
 // ----- Page-Specific Styles ----- //
 
-#digital-subscription-checkout-page-int,
-#digital-subscription-checkout-page-uk,
-#digital-subscription-checkout-page-us,
-#digital-subscription-checkout-page-au {
```

Comment:
i wonder if we gain an advantage from this language-id thing in _any_ page, seems fiddly to me, not sure if we could remove it across the board while keeping the localised url